### ISSUE TITLE
Notify about new translation suggestions

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-ops/localization#29]
A translation supervisor should get an email notification when there are new translation suggestions in his or her group.

Recommendations:
* Add a Rules module event for "A translation suggestion is submitted" or...
* Add a custom module for the notification about new translation suggestions.

**Comment by olafgrabienski** [comment]
To discuss: Who exactly should get the message? There are three options:

- The **author of the group node** which is also the "Manager". AFAIK, this the only 'unique' person in a group, and he or she can't leave the group unless the group author doesn't change. The group author gets also a notification about new group members.
- All people with the group role **administrator member** in the respective group.
- All people with the group role **translations supervisor** in the respective group.

**Comment by olafgrabienski** [comment]
Regarding the implementation, I was hoping that the *Rules* module provides an event like "User has made a translation suggestion" but that's not the case.

@jenlampton Are you interested to extend *Rules* or to write a custom module for the notification about new translation suggestions?

(Regarding the latter approach maybe of interest: There is a "Translation statistics" block in each group which lists also the number of "suggestions awaiting approval". On "Translate" pages like https://localize.backdropcms.org/translate/languages/de/translate it's possible to filter the suggestion status: *Has no / Has / Is suggestion*.)

**Comment by olafgrabienski** [comment]
A completely different idea would be to not solve this technically on the Localization site but to discuss translation questions somewhere else, e.g. using external tools -- cf. https://github.com/backdrop-ops/localization/issues/7#issuecomment-410776691 -- or the Backdrop forums -- cf. https://github.com/backdrop-ops/localization/issues/7#issuecomment-412093628.

In this case, notifications about new translation suggestions would also be organized by the tool chosen by each individual translation group.

**Comment by olafgrabienski** [comment]
> A completely different idea would be to not solve this technically on the Localization site but to discuss translation questions somewhere else (...)

At the moment, there is however the option (for group members) to suggest translations on the Localization site. People are starting to use this option. We need to inform people who can confirm the suggestions.

**Comment by jenlampton** [comment]
> On "Translate" pages like localize.backdropcms.org/translate/languages/de/translate it's possible to filter the suggestion status: Has no / Has / Is suggestion.)

This should already be possible:
https://localize.backdropcms.org/translate/languages/de/translate?project&status=32&release=all&search=&author&context=all&limit=10&sid=0

Click the "Reveal more filters" link at the lower left of the filters area:
<img width="1996" alt="reveal" src="https://user-images.githubusercontent.com/397895/58825877-3b9a6780-85f4-11e9-98eb-4524db65f72a.png">

Then it's the second drop-down under "Status"
<img width="1998" alt="more" src="https://user-images.githubusercontent.com/397895/58825887-405f1b80-85f4-11e9-8d66-346fcd92a7a2.png">

> We need to inform people who can confirm the suggestions.

@olafgrabienski can you please provide links to some of the pages where these things are happening? I want to be sure I know what you are asking for. Let's start with these:

1) A page with `new translation suggestions` block
2) A page with the "Translation statistics" block
3) A page that shows the author administrative member or translation supervisor for a group or language

**Comment by olafgrabienski** [comment]
>> On "Translate" pages like localize.backdropcms.org/translate/languages/de/translate it's possible to filter the suggestion status: Has no / Has / Is suggestion.)

> This should already be possible: (...)

Yes, I just mentioned it because it might be of interest where the information is already available.

**Comment by olafgrabienski** [comment]
> can you please provide links to some of the pages where these things are happening? I want to be sure I know what you are asking for.

Sure! I was asking primarily for an *email* notification when there are new translation suggestions in a group. (Still open question: who exactly should get the email.)

Regarding the links to pages:

There's is no "New translation suggestions" block, but there is the "Translation statistics" block, e.g. in the sidebar on the page https://localize.backdropcms.org/translate/languages/fr, and it contains the number of "suggestions awaiting approval". This block is sufficient, in my opinion, so no need for another one.

A page that shows the author, administrative member or translation supervisor for a group or language:

- The author is mentioned in the "Authoring information" tab of the `node/NID/edit` page of a Translation group. To go there, you can use the "Edit group" link in the 'group block', e.g. the block "French translation group / Groupe de traduction français" at the bottom of the sidebar on the page https://localize.backdropcms.org/translate/languages/fr.
- The same block has a members link to a admin page, where the members and their (*Organic Groups*) roles are listed, the path of the page is `group/node/NID/admin/people`. Note that most groups don't have admin members or translation supervisors so far. That's because we're still few people, and in this situation it's sufficient to be the author of the group node to manage a group. When more people join a group, we need to work with the roles provided by *Organic Groups*.
  
  Btw, a person who wants to provide *translation suggestions*, generally doesn't need a particular *Organic Groups* role, it's sufficient to be a "member" of the group. See the roles on `admin/config/group/roles/node/l10n_group` and the permissions on `admin/config/group/permissions/node/l10n_group`
